stl/inc/chrono

```diff
@@ -58,20 +58,18 @@ namespace chrono {
     };
 
 #if _HAS_CXX20
+    template <class _Clock, class = void>
+    struct _Is_clock : false_type {};
+

```

Comment:
Is there a reason we are not defining the variable alias and using that for the struct?

Usually most people only need the `_v` alias rather than the struct nowadays

Reply:
Done (structs are indeed expensive for throughput). I forgot that the `void_t` pattern was applicable to variable templates - ironic, as I was the one who proposed Variable Templates For Type Traits.